src/modules/private.ts

```diff
@@ -515,6 +531,17 @@ export default class Private {
     );
   }
 
+  /**
+   * @description get the apiKeys associated with an ethereumAddress
+   *
+   * @param ethereumAddress the apiKeys are for
+   */
+  async getApiKeys(
+    ethereumAddress: string,
+  ): Promise<{ apiKeys: ApiKeyResponseObject[] }> {
```

Comment:
this is a different kind of response object (compare `NewApiKey` vs `ApiKey` in our api spec)